openmc/geometry.py

```diff
@@ -322,6 +322,19 @@ def get_all_cells(self):
         else:
             return OrderedDict()
 
+    def get_all_filled_cells(self):
```

Comment:
I'm a little confused. According to the docstrings, this would do the same thing as `Geometry.get_all_material_cells`. How does this method differ?